modules/core_project_factory/versions.tf

```diff
@@ -15,16 +15,16 @@
  */
 
 terraform {
-  required_version = ">= 0.13"
+  required_version = ">= 0.13.0"
 
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.50, < 5.0"
+      version = ">= 4.11.0, < 5.0.0"
```

Comment:
@drandell - Is there a reason to bump the minimum version for core_project sub-module?  If not, let's please leave as-is as the sub-module could be used independently of the main module or sub-modules that require 4.5.

FYI: `~> 4.11` is functionally the same as `>= 4.11.0, < 5.0.0`